data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/exception/LogstashParsingException.java

```diff
@@ -0,0 +1,14 @@
+package org.opensearch.dataprepper.logstash.exception;
+
+/**
+ * Exception thrown when {@link org.opensearch.dataprepper.logstash.parser.LogstashVisitor} is unable to convert
```

Comment:
I'm a big fan of `{@link}`, but I suggest removing it here. The `LogstashVisitor` is an internal implementation detail. Also, exceptions should generally not declare which classes use them. It goes the other way: classes specify which exceptions they throw.

I suggest rewording this somewhat.